src/components/app/details/AboutTagEditModal.tsx

```diff
@@ -0,0 +1,128 @@
+import React, { useEffect, useRef, useState } from 'react'
+import { ReactComponent as Close } from '../../../assets/icons/ic-cross.svg'
+import { Drawer, Progressing, showError } from '../../common'
+import TagLabelSelect from './TagLabelSelect'
+import { AboutAppInfoModalProps, TagType } from '../types'
+import { createAppLabels } from '../service'
+import { toast } from 'react-toastify'
+
+export default function AboutTagEditModal({
+    isLoading,
+    appId,
+    onClose,
+    currentLabelTags,
+    getAppMetaInfoRes,
+}: AboutAppInfoModalProps) {
+    const editLabelRef = useRef(null)
+    const [submitting, setSubmitting] = useState(false)
+    const [labelTags, setLabelTags] = useState<TagType[]>(
+        currentLabelTags?.length
+            ? currentLabelTags
+            : [{ key: '', value: '', propagate: false, isInvalidKey: false, isInvalidValue: false }],
```

Comment:
Can we store this in a separate init  as we are using it in  multiple places

Reply:
Done